src/app/services/itinerary.service.ts

```diff
@@ -2,6 +2,7 @@ import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { Itinerary } from '../models/itinerary';
+import { ItineraryUserPage } from '../models/itineraryUserPage';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
import { ItineraryUserPage } from "../models/itineraryUserPage";
```